examples/Digital filtering/FIRuncFilter_runtime_comparison.py

```diff
@@ -0,0 +1,55 @@
+"""Perform a speed comparison for uncertainty.propagate_filter.FIRuncFilter
+ with different value-types of the covariance of theta
+"""
+
+import numpy as np
+import time
+import pandas as pd
+
+from PyDynamic.uncertainty.propagate_filter import FIRuncFilter
+
+def evaluate(signal, filter, case="full/null/none"):
+    # different covariance matrices
+    if case == "full":
+        U_theta = 1e-2 * np.eye(filter_length)
+    elif case == "null":
+        U_theta = np.zeros((filter_length, filter_length))
+    elif case == "none":
+        U_theta = None
+    else:
+        raise NotImplementedError
+    
+    # measure runtime (timeit.timeit is unable to use both local+global variables)
+    start = time.time()
+    y, Uy = FIRuncFilter(signal, sigma_noise, theta, U_theta, blow=None, kind='float')
```

Comment:
`'float'` is actually not a valid input, neither regarding the docstring, nor the code.

```suggestion
    y, Uy = FIRuncFilter(signal, sigma_noise, theta, U_theta, blow=None)
```